includes/class-micropub-endpoint.php

```diff
@@ -77,24 +113,14 @@ protected static function load_input( $request ) {
 		} elseif ( ! $content_type ||
 			'application/x-www-form-urlencoded' === $content_type ||
 			'multipart/form-data' === $content_type ) {
-			static::$input = array();
-			foreach ( $request->get_body_params() as $key => $val ) {
-				if ( 'action' === $key || 'url' === $key ) {
-					static::$input[ $key ] = $val;
-				} elseif ( 'h' === $key ) {
-					static::$input['type'] = array( 'h-' . $val );
-				} elseif ( 'access_token' === $key ) {
-					continue;
-				} else {
-					static::$input['properties']         = self::get( static::$input, 'properties' );
-					static::$input['properties'][ $key ] =
-					( is_array( $val ) && wp_is_numeric_array( $val ) )
-					? $val : array( $val );
-				}
-			}
+				static::$input = self::form_to_json( $request->get_body_params() );
```

Comment:
`form_to_json` is static. does this need to be `static::form_to_json(`?

Reply:
same question with all of the existing `self::get(` calls, since it's static too...?

Reply:
Discussed elsewhere. Yes, PHP is weird.